apps/io-func-sign-issuer/src/infra/http/handlers/create-signature-request.ts

```diff
@@ -68,6 +69,9 @@ export const CreateSignatureRequestHandler = H.of((req: H.HttpRequest) =>
       )
     ),
     RTE.chainW(insertSignatureRequest),
+    RTE.chainFirstW((request) =>
+      pipe(EventName.SIGNATURE_CREATED, createAndSendAnalyticsEvent(request))
+    ),
```

Comment:
I suspect that the order of arguments for `createAndSendAnalyticsEvent` should be swapped. Usually the principle is to put configuration first (`EventName.SIGNATURE_CREATED`) and data (`request`) last. In this case you could have used:

```suggestion
    RTE.chainFirstW(createAndSendAnalyticsEvent(EventName.SIGNATURE_CREATED)),
```

Reply:
see also https://stackoverflow.com/questions/5863128/ordering-of-parameters-to-make-use-of-currying

Reply:
I had already discussed this @lucacavallaro. We decided to take a look at the function `createAndSendAnalyticsEvent` at a later time with @grausof.

Reply:
Right! I would say make this change with a separate PR.